css/main.css

```diff
@@ -1,9 +1,14 @@
 /* this is atan($skew) * $site-width */
+
+/*  -webkit-transform: skewY(-10deg);
```

Comment:
If it's not in use anymore, just remove it. We'll always be able to get it back from the history if necessary.

Reply:
Applies to all other lines where these lines are commented out